internal/hcsoci/layers.go

```diff
@@ -170,14 +146,31 @@ func mountContainerLayers(layerFolders []string, guestRoot string, uvm *uvm.Util
 		return hostedSettings, nil
 	}
 
+	// This is the LCOW layout inside the utilityVM. NNN is the container "number"
+	// which increments for each container created in a utility VM.
+	//
+	// /run/gcs/c/NNN/config.json
+	// /run/gcs/c/NNN/rootfs
+	// /run/gcs/c/NNN/scratch/upper
+	// /run/gcs/c/NNN/scratch/work
+	//
+	// /dev/sda on /tmp/scratch type ext4 (rw,relatime,block_validity,delalloc,barrier,user_xattr,acl)
+	// /dev/pmem0 on /tmp/v0 type ext4 (ro,relatime,block_validity,delalloc,norecovery,barrier,dax,user_xattr,acl)
+	// /dev/sdb on /run/gcs/c/NNN/scratch type ext4 (rw,relatime,block_validity,delalloc,barrier,user_xattr,acl)
+	// overlay on /run/gcs/c/NNN/rootfs type overlay (rw,relatime,lowerdir=/tmp/v0,upperdir=/run/gcs/c/NNN/scratch/upper,workdir=/run/gcs/c/NNN/scratch/work)
+	//
+	// Where /dev/sda      is the scratch for utility VM itself
```

Comment:
This isn't always the case right? /dev/sda isnt always mounted is it?

Reply:
It will be in the 99% case.